botocore/data/endpoints.json

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
Oh can we name it ``_endpoints.json``? We have some scripts where we would have to update them to ``endpoints.json``.